rpc/legacyrpc/methods.go

```diff
@@ -31,6 +31,8 @@ import (
 	"github.com/decred/dcrwallet/wallet"
 	"github.com/decred/dcrwallet/wallet/txrules"
 	"github.com/decred/dcrwallet/wallet/udb"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
```

Comment:
there should be no grpc imports in the json-rpc implementation (and likewise, no json-rpc code in the grpc server)

Reply:
fixed